contracts/UFragmentsPolicy.sol

```diff
@@ -210,6 +210,20 @@ contract UFragmentsPolicy is Ownable {
         rebaseWindowLengthSec = rebaseWindowLengthSec_;
     }
 
+    /**
+     * @notice A multi-chain AMPL interface method. The Ampleforth monetary policy contract
+     *         on the base-chain and XCAmpleController contracts on the satellite-chains
+     *         implement this method. It returns what the current contract believes to be
+     *         the globalAmpleforthEpoch and globalAMPLSupply.
+     * @return globalAmpleforthEpoch The current epoch number.
+     * @return globalAMPLSupply The total supply at the current epoch.
+     */
+    function getGlobalAmpleforthEpochAndAMPLSupply() external view returns (uint256, uint256) {
+        uint256 globalAmpleforthEpoch = epoch;
+        uint256 globalAMPLSupply = uFrags.totalSupply();
```

Comment:
Is this really more efficient than doing...
`return (epoch, uFrags.totalSupply());`

Reply:
👍